otcextensions/sdk/cce/v3/node_pool.py

```diff
@@ -0,0 +1,223 @@
+    count = resource.Body('count', type=int)
+    # Elastic IP address. Dict of {
+    #     type,
+    #     bandwidth:{
+    #        chargemode, size, sharetype
+    #     }
+    # }.
+    floating_ip = resource.Body('eip', type=dict)
+
+
+class TagSpec(resource.Resource):
+    # Properties:
+    # Key of the tag
+    key = resource.Body('key')
+    # Value of the tag
+    value = resource.Body('value')
+
+
+class TaintSpec(resource.Resource):
+    # Properties:
+    # A key can contain letters, digits, hyphens, underscores and periods
+    # up to 63 characters starting with letter or digit.
+    key = resource.Body('key')
+    # A value can contain letters, digits, hyphens, underscores and periods
+    # up to 63 characters starting with letter or digit.
+    value = resource.Body('value')
+    # Effect available options: NoSchedule, PreferNoSchedule, NoExecute
+    effect = resource.Body('effect')
+
+
+class ExtendParamSpec(resource.Resource):
+    # Properties:
+    # Script required before the installation
+    # Input must be Base64 encoded
+    preinstall_script = resource.Body('alpha.cce/preInstall')
+    # Script required after the installation
+    # Input must be Base64 encoded
+    postinstall_script = resource.Body('alpha.cce/postInstall')
+    # ConfigMap of the Docker data disk
+    lvm_config = resource.Body('DockerLVMConfigOverride')
+    # Maximum number of pods on the node
+    max_pods = resource.Body('maxPods', type=int)
+    # ID of a custom node image
+    # Mandatory if custom image is used on a bare metall node
+    node_image_id = resource.Body('alpha.cce/NodeImageID')
+    # Public key of the node.
+    public_key = resource.Body('publicKey')
+
+
+class AutoScalingSpec(resource.Resource):
+    # Enable or disable auto scaling
+    enable = resource.Body('enable', type=bool)
+    # Minimum number of nodes after a scale-up if auto-scaling is enabled.
+    min_node_count = resource.Body('minNodeCount', type=int)
+    # Maximum number of nodes after scale-up, if auto-scaling is enabled.
+    # The value must be equal or greater than min_node_count and cannot
+    # exceed the maximum number of nodes of the cluster.
+    max_node_count = resource.Body('maxNodeCount', type=int)
+    # Node pool weight. A higher weight indicates a higher priority
+    # in scale-up.
+    priority = resource.Body('priority', type=int)
+    # Interval in minutes during which nodes added aafter a scale-up
+    # will not be deleted.
+    scale_down_cooldown_time = resource.Body('scaleDownCooldownTime', type=int)
+
+
+class NodeManagementSpec(resource.Resource):
+    # ECS groupt id of the ECS group to which those nodes belong
+    # after creation.
+    ecs_group_id = resource.Body('serverGroupReference')
+
+
+class SubnetIdSpec(resource.Resource):
+    # ID of the subnet to which the NIC belongs
+    subnet_id = resource.Body('subnetId')
+
+
+class NodeNicSpec(resource.Resource):
+    # Description about the primary Nic
```

Comment:
can you write `Nic` in upper case? eg in 127 line

Reply:
changed